docs/source/contributing/developing.adoc

```diff
@@ -280,6 +280,17 @@ For example, tests can be run against a stopped {project} instance with the imag
 $ make integration_all RUN_BEFORE_FEATURE="start; stop; config set image-caching true"
 ----
 
+COPY_OC_FROM::
+
+To test Minishift with non-default oc binary you can use the `COPY_OC_FROM` parameter.
+It specifies the path to the oc binary which will be copied to the Minishift home folder before every feature run.
```

Comment:
I'd replace "It" with "This parameter" in order to make this sentence more explicit about what it's describing. I'd also use "before each" instead of "before every".

As a side note, instead of mentioning the "Minishift home folder", would it be accurate to refer to `$MINISHIFT_HOME`?